canonical-web-and-design/ubuntu.com
# Fix sass lint errors

There were many sass-lint errors. This fixes them all.

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/9843

QA
--

Notice that the `sass-lint` github action check passes. You can also verify this locally with `dotrun lint-scss`.

Browse the site. Try to find pages on the site that use all of the patterns I touched, if you can. Make sure they all look fine.
